proto/controller.proto

```diff
@@ -42,7 +115,7 @@ message CreateTopologyRequest {
 // Returns create topology response.
 message CreateTopologyResponse {
   string topology_name = 1;
-  NodeState state = 2;
+  TopologyState state = 2;
```

Comment:
we may have to add another field here for the kubecfg (so the topo knows which cluster to create in)

Reply:
Done